lib/player.rb

```diff
@@ -0,0 +1,17 @@
+class Player
+
+  def initialize(board)
+    @board = board
+  end
+
+  def generate_ships
+    place_ship(Ship.new.carrier)
```

Comment:
The implementation of this method is only particular to the Computer player, no?
I guess you haven't implemented how the Human player places a ship yet?

Reply:
@sarahabimay the implementation of the human player placing ships on the board lives in `Display` and `HumanPlayer`